superset/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -17,16 +17,5616 @@ msgid ""
+
+#: superset-frontend/src/explore/controlPanels/sections.jsx:274
+msgid "Pandas resample method"
+msgstr "Méthode de ré-échantillonnage Pandas"
+
+#: superset-frontend/src/profile/components/App.jsx:47
+#: superset-frontend/src/welcome/Welcome.jsx:100
+#: superset-frontend/src/welcome/Welcome.jsx:105
+msgid "Favorites"
+msgstr "Favoris"
+
+#: superset-frontend/src/profile/components/App.jsx:61
+msgid "Created Content"
+msgstr "Contenu créé"
+
+#: superset-frontend/src/profile/components/App.jsx:75
+msgid "Recent Activity"
+msgstr "Activités récentes"
+
+#: superset-frontend/src/profile/components/App.jsx:89
+msgid "Security & Access"
+msgstr "Securité et accès"
+
+#: superset-frontend/src/profile/components/CreatedContent.jsx:53
+msgid "No charts"
+msgstr "Aucun graphique"
+
+#: superset-frontend/src/profile/components/CreatedContent.jsx:70
+msgid "No dashboards"
+msgstr "Aucun tableau de bord"
+
+#: superset-frontend/src/profile/components/Favorites.jsx:54
+msgid "No favorite charts yet, go click on stars!"
+msgstr "Aucun graphique favori pour le moment, cliquer sur les étoiles !"
+
+#: superset-frontend/src/profile/components/Favorites.jsx:71
+msgid "No favorite dashboards yet, go click on stars!"
+msgstr "Aucun tableau de bord favori pour le moment, cliquer sur les étoiles !"
+
+#: superset-frontend/src/profile/components/Security.jsx:57
+msgid "Datasources"
+msgstr "Sources de données"
+
+#: superset-frontend/src/profile/components/UserInfo.jsx:37
+msgid "Profile picture provided by Gravatar"
+msgstr "Image de profil fournie par Gravatar"
+
+#: superset-frontend/src/profile/components/UserInfo.jsx:55
+msgid "joined"
+msgstr "relié"
+
+#: superset-frontend/src/profile/components/UserInfo.jsx:67
+msgid "id:"
+msgstr "id:"
+
+#: superset-frontend/src/visualizations/FilterBox/FilterBox.jsx:180
+msgid "Select start and end date"
+msgstr "Sélectionnez la date de début et de fin"
+
+#: superset-frontend/src/visualizations/FilterBox/FilterBox.jsx:285
+#, python-format
+msgid "Select [%s]"
+msgstr "Selectionner [%s]"
+
+#: superset-frontend/src/visualizations/FilterBox/FilterBoxChartPlugin.js:25
+msgid "Filter Box"
+msgstr "Boite de filtre"
+
+#: superset-frontend/src/visualizations/TimeTable/TimeTableChartPlugin.js:25
+msgid "Time-series Table"
+msgstr "Tableau des séries temporelles"
+
+#: superset-frontend/src/welcome/DashboardTable.jsx:125
+msgid ""
+"You don't have the necessary permissions to load dashboards. Please contact "
+"your administrator."
+msgstr ""
+"Vous n'avez pas les autorisations nécessaires pour charger les tableaux de "
+"bord. Veuillez contacter votre administrateur."
+
+#: superset-frontend/src/welcome/DashboardTable.jsx:131
+msgid "An error occurred while fetching Dashboards"
+msgstr "Une erreur s'est produite lors de la récupération des tableaux de bord"
+
+#: superset-frontend/src/welcome/Welcome.jsx:87
+#: superset-frontend/src/welcome/Welcome.jsx:92
+msgid "Recently Viewed"
+msgstr "Vu récemment"
+=======
```

Comment:
This looks like a merge artifact still :/ maybe something got messed up when resolving conflicts?

Reply:
yes, but it should be ok now :)